cmd/vic-machine/configure/configure.go

```diff
@@ -184,6 +257,10 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 			v.Network.Nameservers = n.ExecutorConfig.Networks[k].Network.Nameservers
 		}
 	}
+	o.HostCertificate = n.HostCertificate
```

Comment:
I will add a functional test which does not provide any certificate options, thus leaving this field `nil`, and make sure that the VCH is left unmodified and the certificate is not removed. If what you're saying is correct I'll wrap this in a `if n.HostCertificate != nil` and in any case we should have the test described above so it's a useful exercise regardless and it will be useful to have that test running.